storage/migrate.go

```diff
@@ -0,0 +1,155 @@
+package storage
+
+import (
+	"context"
+	"strconv"
+
+	_ "github.com/filecoin-project/sentinel-visor/storage/migrations"
+	"github.com/go-pg/migrations/v8"
+	"github.com/go-pg/pg/v10"
+	"golang.org/x/xerrors"
+)
+
+// GetSchemaVersions returns the schema version in the database and the latest schema version defined by the available
+// migrations.
+func (d *Database) GetSchemaVersions(ctx context.Context) (int64, int64, error) {
```

Comment:
nit: could use an `int` and avoid the casting below

Reply:
done